src/app/tests/TestReadInteraction.cpp

```diff
@@ -143,79 +246,10 @@ void TestReadInteraction::GenerateReportData(nlTestSuite * apSuite, void * apCon
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 }
 
-void TestReadInteraction::TestReadClient(nlTestSuite * apSuite, void * apContext)
-{
-    CHIP_ERROR err = CHIP_NO_ERROR;
-
-    app::ReadClient readClient;
-    EventNumber eventNumber = 0;
-
-    chip::app::InteractionModelDelegate delegate;
-    System::PacketBufferHandle buf = System::PacketBufferHandle::New(System::PacketBuffer::kMaxSize);
-    err                            = readClient.Init(&gExchangeManager, &delegate, 0 /* application identifier */);
-    NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
-
-    err = readClient.SendReadRequest(kTestDeviceNodeId, gAdminId, nullptr /* apSecureSession */, nullptr /*apEventPathParamsList*/,
-                                     0 /*aEventPathParamsListSize*/, nullptr /*apAttributePathParamsList*/,
-                                     0 /*aAttributePathParamsListSize*/, eventNumber /*aEventNumber*/);
-    NL_TEST_ASSERT(apSuite, err == CHIP_ERROR_NOT_CONNECTED);
-
-    GenerateReportData(apSuite, apContext, buf);
-
-    err = readClient.ProcessReportData(std::move(buf));
-    NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
-
-    readClient.Shutdown();
-}
-
-void TestReadInteraction::TestReadHandler(nlTestSuite * apSuite, void * apContext)
```

Comment:
Why remove this?

Reply:
Accidental, remove, just bring it back, thanks